### PR TITLE
925962-Update the UG documentation for scheduler team control newly implemented features.

### PR DESCRIPTION
**Pull request by sanjayvikramsf4373**
### Description: ###

I have added the UG content for below features:

- Key Navigation Support - SfTreeMap Control
- Text Annotation Background - ImageEditor
- Number of Visible Weeks support - SfScheduler.

### CODE REVIEW

File: wpf/Image-Editor/Text.md [hunk 1]
```diff
@@ -79,6 +83,7 @@ Using the following properties in TextSettings, text can be customized.
 * [`Color`](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.ImageEditor.TextSettings.html#Syncfusion_UI_Xaml_ImageEditor_TextSettings_Color) - Modifies the font color of the text.
 * [`TextAlignment`](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.ImageEditor.TextSettings.html#Syncfusion_UI_Xaml_ImageEditor_TextSettings_TextAlignment) - Text alignment such as left, top, and center can be applied.
 * [`TextEffects`](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.ImageEditor.TextSettings.html#Syncfusion_UI_Xaml_ImageEditor_TextSettings_TextEffects) - Text effects such as bold, italic, and underline can be applied.
+* [`Background`](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.ImageEditor.TextSettings.html) -  Specifies the background color of the text annotation. The default value is Brush.Transparent.
```
Comment (Karthickmani97) [inline wpf/Image-Editor/Text.md:86]: @sanjayvikramsf4373 , No need to add API link now. We can add once vol 4 released. So remove

File: wpf/Image-Editor/Text.md [hunk 2]
```diff
@@ -52,6 +53,9 @@ You can make the text bold, italic, or underline by clicking the corresponding i
 
 Text can be aligned to the left, center, or right by clicking the corresponding icon. By default, text will be aligned at the left.
 
+### Background
+
+The Background of the Text color can be customized.
```
Comment (Karthickmani97) [inline wpf/Image-Editor/Text.md:58]: Background can be customized by using how? add sentence clearly

File: wpf/Maps/Markers.md [hunk 3]
```diff
@@ -128,6 +128,96 @@ The Maps control provides the support for defining the custom markers using the
...
+             } 
+
+             return this.SouthAmericaMarkerTemplate; 
+       } 
+
+       return base.SelectTemplate(item, container); 
+   } 
+ }
+
+    public class ViewModel
```
Comment (Karthickmani97) [inline wpf/Maps/Markers.md:199]: @sanjayvikramsf4373 , Add view model and model in separate subtitle

File: wpf/Maps/Markers.md [hunk 4]
```diff
@@ -128,6 +128,96 @@ The Maps control provides the support for defining the custom markers using the
 
 ![Custom Marker Image](Marker_images/Marker_CustomMarkerImg.png)
 
+## Customizing Marker Templates Using DataTemplateSelector
+
+A DataTemplateSelector can be used to dynamically customize marker appearances. This allows you to apply different templates based on specific conditions or data-bound properties using DataTemplateSelector in MarkerTemplateSelector property.
+
+{% tabs %}
+
+{% highlight xaml %}
```
Comment (Karthickmani97) [inline wpf/Maps/Markers.md:137]: @sanjayvikramsf4373 , follow below format

image

File: wpf/Maps/Markers.md [hunk 5]
```diff
@@ -128,6 +128,96 @@ The Maps control provides the support for defining the custom markers using the
 
 ![Custom Marker Image](Marker_images/Marker_CustomMarkerImg.png)
 
+## Customizing Marker Templates Using DataTemplateSelector
```
Comment (Karthickmani97) [inline wpf/Maps/Markers.md:131]: ## Customize marker appearance using DataTemplate
## Customize marker appearance using DataTemplateSelector

Comment (Karthickmani97) [inline wpf/Maps/Markers.md:131]: You can customize the marker appearance by using the [`MarkerTemplate`](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Maps.ShapeFileLayer.html#Syncfusion_UI_Xaml_Maps_ShapeFileLayer_MarkerTemplate) properties of `Imagerylayer` and `ShapeLayer` in the `SfMaps`.

Change in this line - https://github.com/syncfusion-content/wpf-docs/pull/1588/files#diff-6d618aa79fb86b4d6dad26590194374a5acafb8cac79412c4a651780fbcad757R86

File: wpf/Maps/Markers.md [hunk 6]
```diff
@@ -128,6 +128,96 @@ The Maps control provides the support for defining the custom markers using the
...
+
+A DataTemplateSelector can be used to dynamically customize marker appearances. This allows you to apply different templates based on specific conditions or data-bound properties using DataTemplateSelector in MarkerTemplateSelector property.
```
Comment (Karthickmani97) [inline wpf/Maps/Markers.md:133]: You can customize the marker appearance by using the `MarkerTemplateSelector` property of `Imagerylayer` and `ShapeLayer` in the `SfMaps`. The `DataTemplateSelector `can choose a `DataTemplate` at runtime based on the value of a data-bound to marker appearance by using the `MarkerTemplateSelector`. It allows you to choose a different data template for each marker, as well as to customize the appearance of a particular marker based on certain conditions.

File: wpf/Maps/Markers.md [hunk 7]
```diff
@@ -128,6 +128,96 @@ The Maps control provides the support for defining the custom markers using the
...
+    public class Model
+    {
+        public string Label { get; set; }
+        public string Longitude { get; set; }
+        public string Latitude { get; set; }
+    }
+{% endhighlight %}
+
+{% endtabs %}
+
```
Comment (Karthickmani97) [inline wpf/Maps/Markers.md:220]: Add note:

The `DataContext` for both the `MarkerTemplate` and `MarkerTemplateSelector` properties of the `ImageryLayer` and `ShapeLayer` in the `SfMap` is set to `CustomDataSymbol`.

Comment (sanjayvikramsf4373) [reply wpf/Maps/Markers.md:262]: Added notes.

File: wpf/Image-Editor/Text.md (hunk 1 above)
Comment (sanjayvikramsf4373) [reply wpf/Image-Editor/Text.md]: removed added API.

File: wpf/Maps/Markers.md (hunk 3 above)
Comment (sanjayvikramsf4373) [reply wpf/Maps/Markers.md]: Added View model and model in separate subtitle

File: wpf/Maps/Markers.md (hunk 5 above)
Comment (sanjayvikramsf4373) [reply wpf/Maps/Markers.md]: Modified content.

File: wpf/Maps/Markers.md (hunk 6 above)
Comment (sanjayvikramsf4373) [reply wpf/Maps/Markers.md]: Modified content.

File: wpf/Scheduler/Month-View.md [hunk 8]
```diff
@@ -244,6 +244,35 @@ this.Schedule.MonthViewSettings.TrailingDaysVisibility = Visibility.Collapsed;
 
 ![WPF Scheduler month view Leading and Trailing Days Visibility](Month-View_Images/wpf-scheduler-leading-and-trailing-days-visibility.png)
 
+## Customize the number of visible weeks
```
Comment (Jeyasri-Murugan) [inline wpf/Scheduler/Month-View.md:247]: Change the number of visible weeks

**Review by Jeyasri-Murugan — APPROVED** [review]

**Pull request merged**